src/styles/main.scss

```diff
@@ -1 +1,84 @@
-// put code here
+.body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  height: 100vh;
+  align-items: center;
+}
+
+
+.stopwatch {
+  height: 80vmin;
+  width: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__center {
+    box-sizing: border-box;
+    position: relative;
+    height: 5vmin;
+    width: 5vmin;
+    border-radius: 100%;
+    background-color: #f6a603;
+  }
+
+  &__seconds-hand {
+    position: absolute;
+    height: 38vmin;
+    width: 1.5vmin;
+    background-color: #2c8000;
+    bottom: 50%;
+    left: 50%;
+    animation: rotate 60s linear infinite running;
+    transform-origin: bottom center;
+    z-index: -1;
+  }
+
+  &__minutes-hand {
+    position: absolute;
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+    bottom: 50%;
+    left: 50%;
+    animation: rotate 3600s steps(60) infinite running;
+    transform-origin: bottom center;
+    z-index: -2;
+  }
```

Comment:
both hands have a lot of similar styles. Move repeated styles either to a mixin or to a common class selector